assets/js/base/components/filter-element-label/index.tsx

```diff
@@ -9,20 +9,27 @@ import Label from '@woocommerce/base-components/label';
  */
 import './style.scss';
 
+interface FilterElementLabelProps {
+	name: string;
+	count: number;
+}
 /**
  * The label for an filter elements.
  *
  * @param {Object} props Incoming props for the component.
  * @param {string} props.name The name for the label.
  * @param {number} props.count The count of products this status is attached to.
  */
-const FilterElementLabel = ( { name, count } ) => {
+const FilterElementLabel = ( {
+	name,
+	count,
+}: FilterElementLabelProps ): JSX.Element => {
 	return (
 		<>
 			{ name }
 			{ Number.isFinite( count ) && (
 				<Label
-					label={ count }
+					label={ count.toString() }
```

Comment:
I know this fixes the type, but I see some usage of this component like:

```
count={ blockAttributes.showCounts ? count : null }
```

I think this may error? We should fix the cases which consume this component, or make it optional.

Reply:
I see what you mean. Those occurrences should also pass `count.toString()` but they're not typed yet. The `Label` component expects a string as the `count`, but it doesn't do any number logic with the count, it just outputs it as inside a HTML element, so should be safe.